runtime/src/lib.rs

```diff
@@ -539,7 +539,21 @@ impl_runtime_apis! {
 			source: TransactionSource,
 			tx: <Block as BlockT>::Extrinsic,
 		) -> TransactionValidity {
-			Executive::validate_transaction(source, tx)
+			// This runtime API has been hacked to censor transactions of a particular type.
+			// Concretely, it censors all `System::remark` transactions.
+			// A more real-world example would be to censor transactions in which nominators
+			// withdraw stake backing your collator, or add nominations to other collators.
+			// But for now, I'm starting with a simpler example.
```

Comment:
But this is in the runtime. So this means that your consensus censors remarks and not individual nodes.

Reply:
Thanks for taking a look. Yeah, my original thought was that the hacking should not happen in the runtime. I set out initially to hack the transaction pool. Then I was reminded that the TX pool actually does not make any decisions about what transactions are valid or invalid or how they should be prioritized; it calls into the runtime for that.

I totally agree that this hacked code shouldn't end up on-chain as the official canonical code. Rather, I intend to run only the dishonest nodes using this runtime. A hacky but simple way would be to compile it without bumping the spec version and use `--execution native`. A nicer way that I see in the node's help page, but haven't tried yet is using `--wasm-runtime-overrides`.

Reply:
:ok_hand: